app/src/main/java/org/mifos/mobilebanking/utils/MFDatePicker.java

```diff
@@ -97,6 +99,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
                 dialog.getDatePicker().setMaxDate(new Date().getTime());
                 break;
         }
+
```

Comment:
Irrelevant space . Kindly remove it @abhin2149

Reply:
okay

Reply:
There will be an option to solve conflicts. Click it and you know where the files are conflicting.